backend/kernelCI/settings.py

```diff
@@ -35,14 +35,28 @@ def get_json_env_var(name, default):
 # See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure--!70an0r@i00)oqf!3uq_)9dx2^%)xs+(ade0aie+l#6*rh-%#"
+SECRET_KEY = get_json_env_var(
+    "DJANGO_SECRET_KEY", "django-insecure--!70an0r@i00)oqf!3uq_)9dx2^%)"
+)
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = get_json_env_var('DEBUG', True)
+DEBUG = False
+
+ENV_DEBUG = get_json_env_var("DEBUG", False)
+
+if ENV_DEBUG is True or ENV_DEBUG == "True" or ENV_DEBUG == "true":
```

Comment:
It would be nice to create a `isTruly` function to reuse this logic when we need it.